libparsec/crates/client/src/connection_monitor.rs

```diff
@@ -3,7 +3,7 @@
 use std::{ops::ControlFlow, sync::Arc};
 
 use libparsec_client_connection::{
-    AuthenticatedCmds, ConnectionError, RateLimiter, SSEResponseOrMissedEvents,
+    AuthenticatedCmds, ConnectionError, RateLimiter, SSEId, SSEResponseOrMissedEvents,
```

Comment:
Call it `SSEEventID` ? given it is closely related to `SSEEvent`

(and with capital `ID` given that's what we do in `VlobID`, `EntryID` etc.)

Reply:
done